src/main/scala/cromwell/engine/backend/jes/OperationMetadata.scala

```diff
@@ -14,10 +16,19 @@ object OperationMetadata {
       metadata.get("endTime").map(_.asInstanceOf[String].toDate))
   }
 
-  val Iso8601DateFormat = new SimpleDateFormat("yyyy-MM-dd'T'HH:mm:ss.SSSXXX")
+  val expectedDateFormat = "yyyy-MM-dd'T'HH:mm:ss.SSSXXX"
+  val defaultDate = Date.from(Instant.ofEpochMilli(0))
 
   implicit class EnhancedString(val string: String) extends AnyVal {
-    def toDate: Date = Iso8601DateFormat.parse(string.asInstanceOf[String])
+    def toDate: Date = try {
+      val Iso8601DateFormat = new SimpleDateFormat(expectedDateFormat)
+      Iso8601DateFormat.parse(string.asInstanceOf[String])
```

Comment:
Right, ok.  It still makes me nervous that this API is unexpectedly stateful and this code could be refactored or copy/pasted into a situation that isn't thread safe.  So even though this particular case is fine, perhaps a threadsafe library like Joda might be preferable.